clinica/iotools/utils/data_handling_cli.py

```diff
@@ -156,6 +175,29 @@ def run_command(self, args):
         else:
             makedirs(args.output_bids_directory)
 
-        center_all_nifti(args.bids_directory, args.output_bids_directory)
-        cprint(Fore.GREEN + 'All NIfTI files/images of BIDS folder ' + abspath(args.bids_directory)
-               + ' have been centered in output folder ' + abspath(args.output_bids_directory) + Fore.RESET)
+        split_modality = args.modality.split(' ')
+        # Remove empty str in list
+        split_modality = [element for element in split_modality if element]
+
+        cprint('Clinica is now centering the requested images.')
+        centered_files = center_all_nifti(abspath(args.bids_directory),
+                                          abspath(args.output_bids_directory),
+                                          split_modality,
+                                          center_all_files=args.center_all_files)
+
+        # Write list of created files
+        timestamp = time.strftime("%Y%m%d-%H%M%S")
```

Comment:
If we want be homogenous with:
```python
        timestamp = strftime('%Y%m%d_%H%M%S', localtime(time()))
        tsv_file = '%s_subjects_sessions_list.tsv' % timestamp
```
in [get_subject_session_list](https://github.com/arnaudmarcoux/clinica/blob/improve_center_nifti/clinica/utils/io.py) function, you would need to change the formatting either in your function or [get_subject_session_list](https://github.com/arnaudmarcoux/clinica/blob/improve_center_nifti/clinica/utils/io.py).

Reply:
Done